test/README.rst

```diff
@@ -49,3 +51,23 @@ Run the test:
     .. code-block:: bash
 
         py.test test
+
+Testing against a Mock engine or a real Snowflake engine
```

Comment:
What is the reason to introduce the mock engine? To introduce one configuration in which tests do not have to really connect snowflake?

Reply:
The basic idea was to be able to test if generated SQL statements are valid, without the need to actually connect to Snowflake. The concept of a mock engine which does nothing but echoing the generated statements is a standard concept in SQLAlchemy, see https://docs.sqlalchemy.org/en/14/core/engines.html#sqlalchemy.create_mock_engine.

Anyway, I'll remove the mock tests if it makes things too complicated.